worlds/minecraft/docs/minecraft_en.md

```diff
@@ -47,16 +47,29 @@ When the console tells you that you have joined the room, you're all set. Congra
 multiworld game! At this point any additional minecraft players may connect to your forge server. To start the game once
 everyone is ready use the command `/start`.
 
-## Manual Installation
+## Non-Windows Installation
+
+The Minecraft Client will install forge and the mod for other operating systems but Java has to be provided by the
+user. Head to [minecraft_versions.json on the MC AP GitHub](https://raw.githubusercontent.com/KonoTyran/Minecraft_AP_Randomizer/master/versions/minecraft_versions.json)
+to see which java version is required. New installations will default to the topmost "release" version.
+- Install the matching Amazon Corretto JDK
+    - see [Manual Installation Software Links](#manual-installation-software-links)
+    - or package manager provided by your OS / distribution
+- Open your `host.yaml` and add the path to your Java below the `minecraft_options` key
+    - `  java: "path/to/java-xx-amazon-corretto/bin/java"`
+- Run the Minecraft Client and select your .apmc file
+
+## Full Manual Installation
 
 It is highly recommended to ues the Archipelago installer to handle the installation of the forge server for you.
-support will not be given for those wishing to manually install forge. For those of you who know how, and wish to do so,
+Support will not be given for those wishing to manually install forge. For those of you who know how, and wish to do so,
 the following links are the versions of the software we use.
 
-### Manual install Software links
+### Manual Installation Software Links
 
 - [Minecraft Forge Download Page](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.18.2.html)
```

Comment:
We should probably also remove the reference to a particular version of Forge as well, even if just in URL. Kono's `minecraft_versions.json` specifies the verified version of forge as well.